client/object_replicate.go

```diff
@@ -226,6 +240,29 @@ func (x *replicateResponse) FromGRPCMessage(gm grpc.Message) error {
 	}
 
 	x.err = apistatus.ErrorFromV2(st)
+	if x.err != nil {
+		return nil
+	}
+
+	if !x._sigRequested {
+		return nil
+	}
+
+	sigGrpc := new(refsgrpc.Signature)
```

Comment:
we can unmarshal `sigV2` directly

Reply:
oh, right. this code was initially for `proto/refs` dir, but it is impossible to use it for known reasons. fixed